### DISCUSSION TITLE
Can't read the out-going Email

### DISCUSSION THREAD

**Discussion by sawmj** [chatwoot/chatwoot#6937] [Q&A] [answered]
Trying to use letter opener to open sent emails but it doesn't work
already followed the doc and added in the `.env` file the following line `LETTER_OPENER=true`

but can't find any emails in tmp folder

can anyone tell me how is this supposed to work ?

Self-hosted on latest ubuntu

**Comment by pranavrajs** [comment, marked as answer]
The letter opener is used only for development. Please use an SMTP service as listed here. 
https://www.chatwoot.com/docs/self-hosted/configuration/environment-variables#configure-emails
